pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.willowtreeapps</groupId>
     <artifactId>conductor-mobile</artifactId>
-    <version>0.9.0</version>
+    <version>0.10.0</version>
```

Comment:
Since this is a bug-fix, 0.9.1 makes more sense, unless this breaks something?

Reply:
there have been 2 other features already merged in which are not released yet (sauce labs parallelization and the platform ID annotations). This would all be released together in 0.10.0. Does that work for you?

Reply:
👍 in reality I think the deployment system on jitpack ignores this version anyway ;)